services/src/main/scala/cromwell/services/metadata/impl/builder/MetadataBuilderActor.scala

```diff
@@ -335,31 +343,41 @@ class MetadataBuilderActor(readMetadataWorkerMaker: () => Props)
     stay()
   }
 
-  def buildAndStop(query: MetadataQuery, eventsList: Seq[MetadataEvent], expandedValues: Map[String, JsValue], target: ActorRef, originalRequest: MetadataReadAction) = {
+  def buildAndStop(query: MetadataQuery, eventsList: Seq[MetadataEvent], expandedValues: Map[String, JsValue], req: ActorRef, originalRequest: MetadataReadAction) = {
```

Comment:
Is there any reason to rename a meaningful `target` to a quite meaningless `req`?
This applies to all that goes below too.

Reply:
the `req` is a requester and this was added deliberately. The recent PR to develop branch made me to distinguish buildAndStop behavior in case of requester presented, so that buildAndStop and processMetadataResponse can take only `target` as it was originally. Great point indeed